tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/services/ClientServiceDeclTest.java

```diff
@@ -0,0 +1,421 @@
+/*
```

Comment:
rename to `ClientResourceAccessActionNegativeTest` and move this to action folder. move tests files accordingly